R/bs-shiny-theme.R

```diff
@@ -0,0 +1,48 @@
+#' Obtain a list of all available built-in \pkg{bslib} themes.
+#'
+#' @param version the major version of Bootstrap.
+#' @param full_path whether to return a path to the installed theme.
+#' @export
+#' @return a character vector of built-in themes provided by \pkg{bslib}.
```

Comment:
Can we rename this file to be something around "builtin"?

Reply:
Yeah good catch, thank you, will do

Reply:
I renamed the two preset themes files:

* `R/bs-theme-preset-builtin.R` (this file)
* `R/bs-theme-preset-bootswatch.R` (previously `R/bootswatch.R`)